_layouts/home.html

```diff
@@ -74,16 +74,17 @@
     <p>We would like to thank all contributors to our Galaxy courses, especially those mentioned in the <a href="CONTRIBUTORS.md">Contributors list</a>, the Galaxy community for their constant support and our funding partners.</p>
 
     <p>
-    <a href="https://www.denbi.de/"><img src="https://raw.githubusercontent.com/bgruening/rbc_docs/master/logo/deNBI_Logo_rgb.png" width="15%" style="max-width:100%;"></a>  
-    <a href="http://www.sfb992.uni-freiburg.de/"><img src="https://raw.githubusercontent.com/bgruening/presentations/bce348bb606c312d531c479e63a66efc2bc38d44/shared/resources/img/MEDEP.jpg" width="15%" style="max-width:100%;"></a>  
-    <a href="https://www.ie-freiburg.mpg.de"><img src="https://raw.githubusercontent.com/bgruening/presentations/master/shared/resources/img/14_MPI_IE_logo_mit_180.gif" width="15%" style="max-width:100%;"></a>  
-    <a href="https://www.uni-freiburg.de/"><img src="https://raw.githubusercontent.com/bgruening/presentations/a2e38e4b007994af798320db3a0131c4bb891c0e/shared/resources/img/logo_freiburg.jpg" width="15%" style="max-width:100%;"></a>
-    <br>
-    <a href="https://wiki.galaxyproject.org/Teach/GTN"><img src="./shared/images/GTNLogo1000.png" width="15%" style="max-width:100%;"></a>
-    <a href="https://www.france-bioinformatique.fr/"><img src="https://www.france-bioinformatique.fr/sites/default/files/ifb-logo_1.png" width="15%" style="max-width:100%;"></a>
+      <a href="https://www.denbi.de/"><img src="https://raw.githubusercontent.com/bgruening/rbc_docs/master/logo/deNBI_Logo_rgb.png" width="15%" style="max-width:100%;"></a>
+      <a href="http://www.sfb992.uni-freiburg.de/"><img src="https://raw.githubusercontent.com/bgruening/presentations/bce348bb606c312d531c479e63a66efc2bc38d44/shared/resources/img/MEDEP.jpg" width="15%" style="max-width:100%;"></a>
+      <a href="https://www.ie-freiburg.mpg.de"><img src="https://raw.githubusercontent.com/bgruening/presentations/master/shared/resources/img/14_MPI_IE_logo_mit_180.gif" width="15%" style="max-width:100%;"></a>
+      <a href="https://www.uni-freiburg.de/"><img src="https://raw.githubusercontent.com/bgruening/presentations/a2e38e4b007994af798320db3a0131c4bb891c0e/shared/resources/img/logo_freiburg.jpg" width="15%" style="max-width:100%;"></a>
+      <br>
+      <a href="https://wiki.galaxyproject.org/Teach/GTN"><img src="./shared/images/GTNLogo1000.png" width="15%" style="max-width:100%;"></a>
+      <a href="https://www.france-bioinformatique.fr/"><img src="https://www.france-bioinformatique.fr/sites/default/files/ifb-logo_1.png" width="15%" style="max-width:100%;"></a>
     </p>
-
-    <p>You want to help us on this project? Please, see the <a href="CONTRIBUTING.md"><code>CONTRIBUTING</code></a> file.</p>
```

Comment:
This line linking to CONTRIBUTING.md has been left out in the reorganisation, worth to save it?